source/lubeck.d

```diff
@@ -1960,6 +1960,94 @@ struct QRResult(T)
     {
         return qrSolve!(allowDestroy)(matrix, tau, b);
     }
+    
+    /++
+    Extract the Q matrix
+    +/
+    auto Q(Flag!"allowDestroy" allowDestroy = No.allowDestroy)
+    {
+        auto work = [T.sizeof * matrix.length].uninitSlice!T;
+        
+        auto a = matrix.slice;
+        auto m = (allowDestroy && a._stride!1 == 1) ? a.assumeCanonical : a.as!T.slice.canonical;
```

Comment:
`a.as!T.slice` makes a copy if the copy.
Well, for stuff like Q, R we can just make a single copy and don't care a lot.

Reply:
I just need one copy. I'm removed the extra.